lib/spec.js

```diff
@@ -75,6 +75,376 @@ templates = [
 
 // @TODO: register rules in each checker!
 rules = {
+<<<<<<< HEAD
```

Comment:
whoops 😀

Reply:
@kirrg001 Sorted that, still failing though 😢